app/assets/stylesheets/components/_navbar.scss

```diff
@@ -0,0 +1,74 @@
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  // position: relative;
+  width: 95%;
+  margin: 0 auto;
+  margin-top: 10px;
+  height: 70px;
+  padding: 16px;
+  background-color: rgb(232, 230, 230);
+  border-radius: 15px;
+
+  a{
+    font-family: 'roboto', sans-serif;
+    text-decoration: none;
+    color: #494949;
+    font-size: 18px;
+    font-weight: 300;
+    transition: all 0.5s ease-in-out;
+
+    &::before {
+      content: attr(data-item);
+      transition: 0.5s;
+      color: #55679b;
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      width: 0;
+      overflow: hidden;
+      &:hover {
+        &::before {
+        width: 100%;
+        transition: all 0.5s ease-in-out;
+        }
```

Comment:
Add an empty line between overflow and &:hover and indent width and transition please